### PR TITLE
Added support to allow restriction to multiple domains

### PR DESCRIPTION
**Pull request by kcharwood**

### CODE REVIEW

File: app/controllers/invite_controller.rb [hunk 1]
```diff
@@ -21,8 +21,13 @@ def submit
     last_name = params[:last_name]
 
     if ENV["RESTRICTED_DOMAIN"]
-      if email.split("@").last != ENV["RESTRICTED_DOMAIN"]
-        @message = "Sorry! Early access is currently restricted to people within the #{ENV["RESTRICTED_DOMAIN"]} domain."
+      domains = ENV["RESTRICTED_DOMAIN"].split(",")
+      if domains.include? email.split("@").last == false
```
Comment (KrauseFx) [inline app/controllers/invite_controller.rb]: Please change to 

```
unless domains.include?(email.split("@").last)
```

Comment (kcharwood) [inline app/controllers/invite_controller.rb]: Will do!

**Comment by kcharwood** [comment]
Updated @KrauseFx !

**Comment by KrauseFx** [comment]
This is great, thanks for your contribution @kcharwood 👍

**Comment by kcharwood** [comment]
No problem 🍻

**Pull request merged**